app/recipe/serializers.py

```diff
@@ -0,0 +1,36 @@
+from rest_framework.serializers import ModelSerializer
+from .models import Recipe, Ingredient
+
+RECIPE_INGREDIENTS = 'ingredients'
+
+class IngredientSerializer(ModelSerializer):
+    class Meta:
+        model = Ingredient
+        fields = ('id', 'name')
+
+
+class RecipeSerializer(ModelSerializer):
+    ingredients = IngredientSerializer(many=True)
+    class Meta:
+        model = Recipe
+        fields = ('id', 'name', 'description', RECIPE_INGREDIENTS, 'created_at')
+        read_only_views = ('id',)
+
+    def create(self, validated_data):
+        ingredients = validated_data.get(RECIPE_INGREDIENTS)
+        del validated_data[RECIPE_INGREDIENTS]
```

Comment:
```suggestion
        ingredients = validated_data.pop(RECIPE_INGREDIENTS, None)
```

Reply:
I didn't know it could be done this way, thanks!